torchci/pages/test/insights.tsx

```diff
@@ -17,9 +17,47 @@ const LASTEST_N_RUNS = 50;
 const ROW_HEIGHT = 240;
 const TO_GB = 1024 * 1024 * 1024;
 
+function formatDate(
+  timestamp: string
+) {
+  // TODO: This is a weird hack, see how to get rid of this later
+  return new Date(timestamp).toISOString().replace('T', ' ').substring(0, 19);
```

Comment:
🤣🤣🤣 Pretty sure there's an existing library we use that can do more prettily, but I'm okay with leaving this as a TODO for now